stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/GapDecorator.kt

```diff
@@ -22,40 +22,34 @@ internal class GapDecorator : BaseDecorator() {
 
     override fun decorateDeletedMessage(
         viewHolder: MessageDeletedViewHolder,
-        data: MessageListItem.MessageItem,
-        isThread: Boolean
+        data: MessageListItem.MessageItem
```

Comment:
The PR removes a bunch of commas in multiple files, making for extra line changes, but those could actually remain there as trailing commas

![image](https://user-images.githubusercontent.com/12054216/105699520-c6d21a80-5f07-11eb-8ab5-2dacfaacfaae.png)

Reply:
It how refactoring in AndroidStudio works)

Reply:
Haha, I figured that that's what happened, that the "remove parameter" refactoring didn't format the files afterwards. Kind of weird that that doesn't happen automatically.

Reply:
It requires tons of manual editing of sources. Since it isn't a strong violation or mistake I'd offer to keep it as is. What do you think? @zsmb13